src/main/resources/anniv5Resources/summaries/ratings.txt

```diff
@@ -12,6 +12,7 @@ BitingColdPack	4	2	2	3	3
 BlacksmithsPack	3	3	4	4	4
 BoardGamePack	4	3	2	4	2
 BulwarkPack	2	5	1	4	4
+CalamityPack	4	2	2	3	4
```

Comment:
From my experience playtesting. The pack is good offensively, has some defense and support (but a bit more niche or at higher rarities), is fine on frontload (the common attacks are pretty good), and provides strong scaling options.